packages/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -52,10 +52,12 @@ export class TriggerEvent {
     private processSubscriber: ProcessSubscriber,
     private integrationRepository: IntegrationRepository,
     private getNovuIntegration: GetNovuIntegration,
+    @Inject(forwardRef(() => EventsPerformanceService))
     protected performanceService: EventsPerformanceService,
     private jobRepository: JobRepository,
     private notificationTemplateRepository: NotificationTemplateRepository,
     private logger: PinoLogger,
+    @Inject(forwardRef(() => AnalyticsService))
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.